hs_docker_base

```diff
@@ -1 +0,0 @@
-Subproject commit c6dd4890293ac3bcec39f14859b1ae749350bd4a
```

Comment:
@mjstealey After merging with latest develop, got this diff from github. Can you take a look to make sure this will not pose any issue once merging this into develop? Also, I made a bit of further changes to hsctl to make sure celery worker to be able to run tasks after running ```hsctl restart``` or running ```hsctl stop``` followed by ```hsctl start``` Please take a quick look to make sure the changes look good.